src/test/java/com/exasol/cloudwatch/fingerprint/FingerprintExtractor.java

```diff
@@ -0,0 +1,32 @@
+package com.exasol.cloudwatch.fingerprint;
+
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+/**
+ * This class provides a method for extracting the fingerprint from an Exasol JDBC URL.
+ */
+public class FingerprintExtractor {
```

Comment:
Why is this class in test sources?

Reply:
We only need this for running the integration tests.